### ISSUE TITLE
Session hardware_id not found.  Creating a new session and trying again. 

### ISSUE THREAD

**Issue by taherkaraki** [dgreif/ring#78]
I don't know whats the issue, but I have updated to 4.4.0, removed cached accessory from persist directory, removed from home, started homebridge, rebooted iphone, added camera again.. Still not updating neither the camera snapshop nor motion sensor state. 

I enabled debug mode and the error i'm getting is: **Session hardware_id not found.  Creating a new session and trying again.** 

Then after a while I get: **Session response rate limited. Waiting to retry for 179 seconds.**

Kindly help.

**Comment by dgreif** [comment]
Sorry to hear you are having issue.  Was everything working on a previous version?  You can try using a refresh token by following the steps in https://github.com/dgreif/ring/wiki/Two-Factor-Auth.  I would also check to make sure your Ring subscription is not expired.  Let me know if any of that helps or if you are still having issues.

**Comment by taherkaraki** [comment]
I'm not using 2FA. It used to be a hit and miss for me on previous version (4.2.3) since I have the 1st gen doorbell. But on 4.4.0 it does not work at all and just produces the above error.

**Comment by taherkaraki** [comment]
Ok my bad. After my reply I investigated everything and found the issue. I have my ring doorbell connected to another router with 4G internet plan since it is faster. But that router had no internet connection and needed rebooting. It didn't ring a bell for me at first since it coincided with the 4.4.0 upgrade and since the error I got didn't lead me in the right direction. Sorry for bothering you man. Keep up the great work.
